keyboards/dztech/dz65rgb/keymaps/default/keymap.c

```diff
@@ -0,0 +1,41 @@
+#include QMK_KEYBOARD_H
+extern bool g_suspend_state;
+#define _LAYER0 0
+#define _LAYER1 1
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+		[_LAYER0] = LAYOUT_65_ansi( /* Base */
+			KC_GESC,       KC_1,     KC_2,     KC_3,  KC_4,  KC_5,  KC_6,    KC_7,  KC_8,    KC_9,     KC_0,     KC_MINS,  KC_EQL,  KC_BSPC,   KC_HOME,\
```

Comment:
The backslashes at the end of these lines are not necessary. You can keep them if you want, but the purpose of them is actually to allow formatting long `#defines` over multiple lines, such as in the `LAYOUT_*` definitions.